assets/js/modules/search-console/components/common/AnalyticsStats.js

```diff
@@ -119,7 +122,7 @@ export default function AnalyticsStats( props ) {
 			: null
```

Comment:
Same here, we need to avoid selecting `getPropertyID` in a view-only context.

Reply:
Updated, thank you!